packages/internal/src/utils.js

```diff
@@ -241,6 +252,7 @@ export const deeplyFulfilledObject = obj => {
  * @returns {<T>(fn: () => Promise<T>) => Promise<{ result: T, duration: number }>}
  */
 export const makeMeasureSeconds = currentTimeMillisec => {
+  /** @param {() => any} fn */
```

Comment:
For this one we should be able to easy propagate through the result by templetizing here
```suggestion
  /**
   * @template {() => any} T
   * @param {T} fn
   */

Reply:
What different will it make? `makeMeasureSeconds` defines the type already.

Reply:
Right, sorry habit of letting inference flow to the return types and not use explicit `@returns`.